### ISSUE TITLE
Template parts loading... Site Editor impossibly slow

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/sb#6]
![image](https://user-images.githubusercontent.com/2474435/112222113-328ee600-8c20-11eb-89f4-adc4d784c8ee.png)

**Comment by bobbingwide** [comment]
Well, I think this particular problem has gone away. 
I do have problems with the Site Editor hanging, with Gutenberg 13.1.0
![image](https://user-images.githubusercontent.com/2474435/168069843-d15a98d5-b08d-4599-835f-2e4b5463b61e.png)
it's painfully slow both locally and in my SiteGround hosting.
![image](https://user-images.githubusercontent.com/2474435/168070076-50e86c9c-2394-4f26-8801-764faf227c57.png)

In my local install I get a WSOD.

**Comment by bobbingwide** [comment]
![image](https://user-images.githubusercontent.com/2474435/168071505-9f801a4a-0597-4503-b273-e363c9288391.png)

30 seconds for a mouse move is a bit much!

**Comment by bobbingwide** [comment]
In my latest tests, with WordPress 6.0 and Gutenberg 13.3.0 the Classic block is producing a slightly different version of the `Uncaught TypeError: Cannot read properties of undefined (reading 'tinymce')` message.  

```
Uncaught TypeError: Cannot read properties of undefined (reading 'tinymce')
    at block-library.js?ver=016c65731f009ca1e5b5974727017d18:5514:29
    at invokePassiveEffectCreate (react-dom.js?ver=17.0.1:23517:22)
    at HTMLUnknownElement.callCallback (react-dom.js?ver=17.0.1:3942:16)
    at Object.invokeGuardedCallbackDev (react-dom.js?ver=17.0.1:3991:18)
    at invokeGuardedCallback (react-dom.js?ver=17.0.1:4053:33)
    at flushPassiveEffectsImpl (react-dom.js?ver=17.0.1:23604:11)
    at unstable_runWithPriority (react.js?ver=17.0.1:2764:14)
    at runWithPriority$1 (react-dom.js?ver=17.0.1:11306:12)
    at flushPassiveEffects (react-dom.js?ver=17.0.1:23477:16)
    at performSyncWorkOnRoot (react-dom.js?ver=17.0.1:22299:5)
react-dom.js?ver=17.0.1:20115 The above error occurred in the <ClassicEdit> component:

    at ClassicEdit (https://s.b/wp55/wp-includes/js/dist/block-library.js?ver=016c65731f009ca1e5b5974727017d18:5487:5)
    at Edit (https://s.b/wp55/wp-includes/js/dist/block-editor.js?ver=39c7981dd22ec80281080455892940e6:12918:5)
...
```

This problem is probably covered by https://github.com/WordPress/gutenberg/issues/23086

The Classic block in question is the `[bw_list]` shortcode in the sequentially-biased.html template part.
``` 
<!-- wp:heading {"level":4}  -->
<h4>Sequentially Biased</h4>
<!-- /wp:heading -->

[bw_list post_type=bigram posts_per_page=16 orderby=date order=DESC exclude=0]
```

This particular instance could be reworked to use the shortcode block or even the query loop.

**Comment by bobbingwide** [comment]
The problem with the Classic block failing has now been resolved in Gutenberg 15.3.1 or WordPress 6.2-beta5.
See https://github.com/WordPress/gutenberg/issues/48938

With the latest changes to the Site Editor the performance is also much better. 

Closing therefore.
